apps/cookbook/src/app/showcase/accordion-showcase/accordion-showcase.component.html

```diff
@@ -34,18 +34,22 @@ <h2>Accordion inside a card</h2>
     <cookbook-accordion-card-example #accordionCardExample></cookbook-accordion-card-example>
   </cookbook-example-viewer>
 
-  <h2>Accordion inside card with items</h2>
+  <h2>Accordion with list</h2>
   <p>
-    Since kirby-items come with their own built-in padding the accordions below have
-    <code>hasPadding</code>
-    set to
-    <code>false</code>
-    to avoid double padding. This allows for content inside items to be aligned with the accordion header.
+    Placing a list within an accordion item will automatically set the correct styling such that the
+    list looks correct without having to adjust the components.
```

Comment:
Consider something like this instead, to give the reader a little more context:
"Placing a list within an accordion item will automatically adapt spacing and strip the necessary list-styling to ensure that the list looks correct without further adjustments."